### PR TITLE
zh translation: docs/tasks/traffic management/ingress/ingress sni passthrough/index.md

### PR DESCRIPTION
**Pull request by orangegzx**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 无 TLS 终止的 Ingress Gateway
-description: 说明了如何为一个 ingress gateway 配置 SNI 透传。
+description: 如何为一个 ingress gateway 配置 SNI 透传。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/index.md:3]:
```suggestion
description: 如何为一个 I￼ngress gateway 配置 SNI 透传。
```

Comment (elkan1788) [reply content/zh/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/index.md:3]: @kebe7jun @orangegzx  建议还是把 `gateway` 统一翻译成网关，比较合适。

Comment (orangegzx) [reply content/zh/docs/tasks/traffic-management/ingress/ingress-sni-passthrough/index.md]: 已修改

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**